virttest/shared/cfg/machines.cfg

```diff
@@ -6,12 +6,14 @@ variants:
         machine_type = pc
         enable_pvpanic = yes
         ioport_pvpanic = 0x505
+        vmcoreinfo = yes
```

Comment:
I suppose by default it should be 'no' so this modification is not necessary, only few test cases need to specifically set it to 'yes'.

Reply:
Hi qianqian, 

There's a background here, our developer advice us with a vmcoreinfo default for our tests, since it's a debug device, we can use it to generate dump file. And also discussed  with lijin and leidong before, they all agree it, so I changed it on this file~ 

Thanks~
Peixiu

Reply:
Thanks @peixiu for the introduction, I think here we need to request @luckyh and @yanan-fu for their advise.

Reply:
Hi @peixiu I think we can remove this global changing  for now and have it in. And later it can be added when need. What do you think?

Reply:
@vivianQizhu I agree it, has updated, thanks~